### ISSUE TITLE
Latest ucto make check fails on Mac OS X

### ISSUE THREAD

**Issue by proycon** [LanguageMachines/ucto#54]
```
$ brew install --debug ucto
```

```
===================================
   ucto 0.13: src/test-suite.log
===================================

# TOTAL: 1
# PASS:  0
# SKIP:  0
# XFAIL: 0
# FAIL:  1
# XPASS: 0
# ERROR: 0

.. contents:: :depth: 2

FAIL: tst.sh
============

libc++abi.dylib: terminating with uncaught exception of type std::runtime_error: TiCC::searchFilesMatch: the name '' doesn't match a file or directory.
```

**Comment by proycon** [comment]
Does not seem to reproduce in the development LaMachine, perhaps an artefact of the homebrew installation?

**Comment by kosloot** [comment]
Fixed in latest release 0.13.2
